### PR TITLE
Configuration for command line options 

### PR DESCRIPTION
**Pull request by lukaszsagol**
Option to configure command line options used when running `browserify` command. Really useful for people who are using transforms when bundling with browserify, but can be handy in many other situations.

Tests included.

### CODE REVIEW

**Comment by hsume2** [comment]
@zgryw thanks for working on this. I just merged a separate PR (https://github.com/hsume2/browserify-rails/pull/12) which is causing conflicts with your PR. I'm guessing the conflicts are mostly in the Readme. Can you rework this PR? And I'll see about getting it merged. Thanks!

File: lib/browserify-rails/browserify_processor.rb [hunk 1]
```diff
@@ -87,5 +80,25 @@ def run_browserify(options)
...
+
+      if config.commandline_options.present?
+        if config.commandline_options.is_a? Array
+          options.push(*config.commandline_options)
+        else
+          options.push(config.commandline_options)
+        end
+      end
+
+      options.uniq.join(' ')
```
Comment (martenlienen) [inline lib/browserify-rails/browserify_processor.rb]: Please use " over ' for consistency

File: lib/browserify-rails/browserify_processor.rb [hunk 2]
```diff
@@ -87,5 +80,25 @@ def run_browserify(options)
...
+
+      options.push("-d") if Rails.env.development?
+
+      if config.commandline_options.present?
+        if config.commandline_options.is_a? Array
+          options.push(*config.commandline_options)
+        else
+          options.push(config.commandline_options)
+        end
+      end
```
Comment (martenlienen) [inline lib/browserify-rails/browserify_processor.rb]: Please refactor this whole block to

``` ruby
options += Array(config.commandline_options)
```

**Comment by lukaszsagol** [comment]
Sure!

Merged master into my branch, made the style fixes mentioned by @CQQL and fixed tests to work with changes introduced by #12

**Comment by hsume2** [comment]
Awesome, this looks great, thanks @zgryw!

**Pull request merged**